tests/parm/v16_c96.nml.IN

```diff
@@ -120,9 +121,11 @@ deflate_level=1
 /
 
 &gfs_physics_nml
-  fhzero       = 6
+  fhzero       = @[FHZERO]
   h2o_phys     = .true.
-  ldiag3d      = .false.
+  lssav          = @[LDIAG3D]
```

Comment:
if `lssav` needs to switch, then it should have its own variable, not use `LDIAG3D`. However, setting `lssav` is a moot point because it is hardcoded to `.true.` in `GFS_time_vary_pre_timestep_init` (ever since IPD), so please revert this change.